apis/Google.Cloud.Logging.Console/Google.Cloud.Logging.Console.Tests/GoogleCloudConsoleFormatterAugmentationTest.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2020 Google LLC
```

Comment:
```suggestion
// Copyright 2025 Google LLC
```

Reply:
But I'd move these tests to GoogleCloudConsoleFormatterTest.cs along with the other tests.